src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposePush.groovy

```diff
@@ -0,0 +1,25 @@
+package com.avast.gradle.dockercompose.tasks
+
+import com.avast.gradle.dockercompose.ComposeSettings
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+class ComposePush extends DefaultTask {
+
+    ComposeSettings settings
+
+    ComposePush() {
+        group = 'docker'
+        description = 'Pushes images for services of docker-compose project'
+    }
+
+    @TaskAction
+    void push() {
+        String[] args = ['push']
+        if (settings.ignorePushFailure) {
+            args += '--ignore-push-failures'
+        }
+        args += settings.startedServices
```

Comment:
I'm not sure about what services should be specified here. I think that we should pass here only the services that have both `build` and `image` specified in the `docker-compose.yml`, according [the documentation](https://docs.docker.com/compose/reference/push/).

Maybe not passing any arguments (so just dropping this line) would make that work automatically.

Reply:
Specified services are the ones you want to push, but can be subset of ones specified in compose files.

I would propose to add setting for this `pushServices`. So you can have extra services in compose file, but only those specified in `pushServices` would be pushed.

Reply:
```Usage: push [options] [SERVICE...]

Options:
    --ignore-push-failures  Push what it can and ignores images with push failures.```